ensime-mode.el

```diff
@@ -505,9 +506,10 @@ A prefix argument will add the report to the kill ring."
                          (insert-file-contents filePath)
                          (buffer-string))))
                  (progn "Not provided")))))
-    (when arg
+    (unless arg
       (kill-new info))
-    (message info)))
+    (message info)
```

Comment:
Sure, but it has to be tomorrow, I need to read first about creating buffers in emacs 😆